packages/core/src/accessanalyzer/vue/iamPolicyChecks.ts

```diff
@@ -672,14 +711,22 @@ function parseCliErrorMessage(message: string): string {
         message.includes('The security token included in the request is expired')
     ) {
         return IamPolicyChecksConstants.InvalidAwsCredentials
+    } else if (
+        message.includes(
+            `Unexpected error occurred. 'AccessAnalyzer' object has no attribute 'check_access_not_granted'`
+        )
+    ) {
+        return 'ERROR: The dependencies of the Python CLI tools are outdated.'
```

Comment:
You may want to indicate which is minimum version they should install for the check to work as expected

Reply:
This error should only occur if we set the minimum version incorrectly in our CLI tool. We should rather ask them to reinstall the tool, and our tool should have the correct dependencies.

Reply:
I see. Thanks!